spec/features/user/user_can_have_role_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
```

Comment:
This was a test that @deborahleehamel wrote that I just re-wrote to make it passing. I will move it before committing.